### DISCUSSION TITLE
Remote Youtube Playlist

### DISCUSSION THREAD

**Discussion by HauptUber** [smplayer-dev/smplayer#393] [Q&A]
If I create a local m3u playlist containing youtube urls - smplayer will play the list.

If I attempt to play a web hosted m3u playlist containing youtube urls - smplayer will not play the list of youtube urls.

Error message in SMPlayer log - from attempting to play remotely hosted m3u playlist containing youtube urls.

[05:03:31:458] Core::changeCurrentSec: mplayer reports that now it's playing
[05:03:31:458] BaseGui::checkStayOnTop
[05:03:32:086] MPVProcess::parseLine: "Failed to recognize file format."
[05:03:32:086] MPVProcess::parseLine: "Playing: https://www.youtube.com/watch?v=YewVugPHon4"

How can I overcome that error ?

**Comment by smplayer-dev** [comment]
Does it work if you load the playlist in the playlist window?

**Reply by HauptUber** [reply]
If the remotely hosted M3U is downloaded - it can be played.

* However * there are several M3U files that are linked together - which would require continuously downloading playlists.

There is a tab for opening a URL - which works for pls / m3u8 files - why would it not allow for the opening up a remote playlist ?

**Reply by smplayer-dev** [reply]
Many m3u8 urls are adaptive videos, not playlists, so smplayer just passes the url to mpv.

**Reply by HauptUber** [reply]
How do I set up the smplayer so that I can enter the URL of the M3U playlist - so I do not have to download it ?

**Reply by smplayer-dev** [reply]
There's an option to load a playlist from a URL in the playlist window.
